src/client/webpack.config.js

```diff
@@ -145,8 +161,9 @@ if (isProductionMode){
     }
     return loader;
   });
+
   webpackConfig.plugins.push(
-    new ExtractTextPlugin('[name].[contenthash].css'),
+    new ExtractTextPlugin('[name].css?[contenthash]'),
```

Comment:
also, we could probably dispose of the whole .css stuff and have it in the bundle. /cc @ZachBray